### PR TITLE
Part 6

### PR DESCRIPTION
**Pull request by Jag50**
Bowling game runs from command prompt

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -93,11 +95,30 @@
...
+                    </archive>
+                </configuration>
 
+            </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
```
Comment (rachelwarburton) [inline pom.xml]: You could try updating it to java 11?

File: src/main/java/bowlingexercise3/BowlingApplication.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    public static void main(String args[]) throws IOException {
+
+        char press, clear;
+        int i = 0;
+
+        while (i < 10) {
+
+            Scanner scanner = new Scanner(System.in);
+
+            System.out.println("Welcome to 10 Pin Bowling!");
```
Comment (rachelwarburton) [inline src/main/java/bowlingexercise3/BowlingApplication.java:18]: It's fine to use System.out.println for now - just be aware that we would always use a logger in production code. If you get time, you could investigate how to replace these statements with a logger.

File: pom.xml [hunk 3]
```diff
@@ -42,26 +42,28 @@
...
             <scope>test</scope>
         </dependency>
+
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>4.13</version>
             <scope>test</scope>
         </dependency>
+
```
Comment (rachelwarburton) [inline pom.xml]: I think you had removed this dependency - maybe it wasn't committed?

File: src/main/java/bowlingexercise3/BowlingApplication.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            }
+
+            System.out.println();
+            System.out.println("Press any key to continue, otherwise enter t to terminate: ");
+
+            press = (char) System.in.read();
+            System.out.println("You Pressed: " + press);
+
+            do {
+                clear = (char) System.in.read();
```
Comment (rachelwarburton) [inline src/main/java/bowlingexercise3/BowlingApplication.java:40]: What happens when you enter two or more sets of scores?

File: pom.xml (hunk 1 above)
Comment (Jag50) [reply pom.xml]: It is on 11. 

    <properties>
        <maven.compiler.source>11</maven.compiler.source>
        <maven.compiler.target>11</maven.compiler.target>
    </properties>

File: pom.xml (hunk 3 above)
Comment (Jag50) [reply pom.xml]: I've updated it now

File: src/main/java/bowlingexercise3/BowlingApplication.java (hunk 4 above)
Comment (Jag50) [reply src/main/java/bowlingexercise3/BowlingApplication.java:40]: They would need to be entered seperately

**Review by rachelwarburton — APPROVED** [review]
looks good to me

**Pull request merged**